com.github.jean28518.Linux-Assistant.yml

```diff
@@ -0,0 +1,40 @@
+app-id: com.github.jean28518.Linux-Assistant
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+
+command: linux-assistant
+finish-args:
+  - "--socket=fallback-x11"
+  - "--device=dri"
+  - "--share=ipc"
+  - "--filesystem=host"
```

Comment:
this is very broad.

Reply:
Linux-Assistant is an administrative tool which reads part of the host filesystem for system recognition and executes a lot commands right on the host.
Also it checks if some applications are installed and some files are present in the host file system.
There is no realistic option omitting that high privilege.